src/oracle/backend.rs

```diff
@@ -24,5 +26,6 @@ impl TypeMetadata for Oracle {
 impl UsesAnsiSavepointSyntax for Oracle {}
 
 // TODO: check if Oracle supports this
-//impl SupportsDefaultKeyword for Oracle {}
+impl SupportsDefaultKeyword for Oracle {}
+impl SupportsReturningClause for Oracle {}
```

Comment:
I don't think Oracle supports the RETURNING clause in a form that diesel expects.

See [Oracle version](https://docs.oracle.com/cd/B19306_01/appdev.102/b14261/returninginto_clause.htm).
See [Postgres version](https://www.postgresql.org/docs/9.5/static/dml-returning.html).